src/core/localization/linksNoTranslation.test.ts

```diff
@@ -0,0 +1,79 @@
+import fs from 'fs';
+import { describe, expect, it, test } from 'vitest';
+import gettextParser from 'gettext-parser';
+import { extractLinkAndLabels } from './extractLinkAndLabelFromMarkdown';
+import type { LinkAndLabel } from './extractLinkAndLabelFromMarkdown';
+
+const languageCodes = ['ar', 'de', 'es', 'id', 'ko', 'uk'];
+
+type GettextTranslation = {
+  msgid: string;
+  msgstr: string[];
+};
+
+type GettextTranslations = {
+  [key: string]: GettextTranslation;
+};
+
+const checkLinksAndLabels = function (
+  engArray: LinkAndLabel[],
+  translatedArray: LinkAndLabel[],
+) {
+  engArray.forEach((engLinkAndLabel: LinkAndLabel, index) => {
+    const translatedLinkAndLabel: LinkAndLabel = translatedArray[index];
+    // If eng version text with link is present and text in translated version is present, verify that translated version has text with link under the same number and generally has text with link.
```

Comment:
This comment is too long (I had to use horizontal scroll), would be better to break it down in multiple lines or make it shorter

Reply:
fixed